hedera/api.py

```diff
@@ -297,6 +301,15 @@ def post(self, request, *args, **kwargs):
 
         return JsonResponse({"data": vl.data()})
 
+    def delete(self, request, *args, **kwargs):
+        data = json.loads(request.body)
+        if "id" in data:
+            count, _ = PersonalVocabularyListEntry.objects.filter(id=data["id"]).delete()
+            if(count != 0):
+                return JsonResponse({"data": True, "id": data["id"]})
+            return HttpResponseNotFound(f"could not find vocab with id={data['id']}")
+        return HttpResponseBadRequest(f"missing 'id'")
```

Comment:
No need for an f string here, if there is no expression/variable to evaluate.

Reply:
ahh should i update it to a json response with the error message as payload?